src/s3.js

```diff
@@ -5,7 +5,7 @@ const knox = require('knox');
 const mime = require('mime-types');
 const promised = require('promised-method');
 
-const WINDOWS_SLASH_REGEX = /\//g;
+const WINDOWS_SLASH_REGEX = /\\/g;
```

Comment:
These seems right to me? Weird that it would be looking for `/` and replacing it with `/`.

Reply:
Just goofed